packages/rich-text/src/RichTextEditor.tsx

```diff
@@ -77,40 +52,32 @@ export const ConnectedRichTextEditor = (props: ConnectedProps) => {
     props.isToolbarHidden && styles.hiddenToolbar
   );
 
-  useEffect(() => {
```

Comment:
This is now done in the `toSlaveValue` hook?

Reply:
The two existing useEffects complements each other AFAIK so both are replaced by `SyncEditorValue`. The normalization bit is replaced by the `normalizeInitialValue` flag + the automatic normalization that happens when using `setEditorContent` function.

`normalizeInitialValue` didn't work for us in the past but I'm giving it a second shot. If it doesn't work then I'd repurpose the `SyncEditorState` to set the initial value as well. There is a trade-off for each solution.